lib/DDG/Spice/SkyscannerFlightSearch.pm

```diff
@@ -0,0 +1,39 @@
+package DDG::Spice::SkyscannerFlightSearch;
+
+# ABSTRACT: This IA will call Skyscanner's API to retrieve the cheapest flight prices to 
+# the chosen destination
+
+use DDG::Spice;
+use strict;
+use warnings;
+
+# Caching - http://docs.duckduckhack.com/backend-reference/api-reference.html#caching
+spice is_cached => 1;
+spice proxy_cache_valid => '200 1d'; # defaults to this automatically
+
+spice wrap_jsonp_callback => 1; # only enable for non-JSONP APIs (i.e. no &callback= parameter)
+spice to => 'http://partners.api.skyscanner.net/apiservices/browseroutes/v1.0/GB/GBP/en-GB/$1/anywhere/anytime/anytime?apikey={{ENV{DDG_SPICE_SKYSCANNER_APIKEY}}}';
```

Comment:
@samheyman thanks for the explanation!

I would suggest we create a mapping of locales -> currencies with a YAML/JSON file that we store in the /share/ directory. Then it can be loaded in the Perl to determine which currency we should use. I suppose we can fallback to USD as it's the most popular currency?

You may be able to find/use another Perl module to map locales in our format to another